app/assets/javascripts/components/reviewer/reviewerDashboard.js.jsx

```diff
@@ -4,17 +4,17 @@ class ReviewerDashboard extends React.Component {
     super(props)
 
     this.cohorts = JSON.parse(this.props.cohorts)
-    this.current_cohort = JSON.parse(this.props.current_cohort)
+    if(this.current_cohort){this.current_cohort = JSON.parse(this.props.current_cohort)}
     this.defineCurrentCohort()
-    this.current_cohort.title = this.current_cohort.title + ' (Current Cohort)'
+    if(this.current_cohort){this.current_cohort.title + ' (Current Cohort)'}
```

Comment:
What does this line do? Should there be an assignment here somewhere?

Reply:
Yup, fixed.